tests/cacl/test_cacl_application.py

```diff
@@ -835,6 +835,123 @@ def generate_scale_rules(duthost, ip_type):
     duthost.command("iptables -I INPUT 3 -p tcp -m tcp --dport 22 -j ACCEPT")
 
 
+def verify_cacl_show_acl_rule(duthost):
+    """
+    Converts the CACL rules in the generated SCALE_ACL_FILE into a dict,
+    and verifies that they match the applied acl rules through the `show acl
+    rule` command.
+    """
+
+    acl_json = duthost.command(f"cat {SCALE_ACL_FILE}")["stdout_lines"]
```

Comment:
@matthew-soulsby  in your scale acl.json file, it only cover DROP forward action, it's better to have DROP and FORWARD mixed rules inside, then the coverage of your case is increased.

Reply:
Good point, I've added a new fixture that generates an acl.json file containing DROP and ACCEPT rules for both IPv4 and IPv6 simultaneously, so it should now cover most possible cases